src/main/java/org/folio/service/transformationfields/TransformationFieldsConfig.java

```diff
@@ -2,25 +2,35 @@
 
 import java.util.Map;
 
+import static org.folio.service.mapping.referencedata.ReferenceDataImpl.ALTERNATIVE_TITLE_TYPES;
 import static org.folio.service.mapping.referencedata.ReferenceDataImpl.IDENTIFIER_TYPES;
+import static org.folio.service.mapping.referencedata.ReferenceDataImpl.INSTANCE_TYPES;
+import static org.folio.service.mapping.referencedata.ReferenceDataImpl.LOAN_TYPES;
+import static org.folio.service.mapping.referencedata.ReferenceDataImpl.MATERIAL_TYPES;
+import static org.folio.service.mapping.referencedata.ReferenceDataImpl.MODES_OF_ISSUANCE;
 
 /**
  * Initial data for the transformation field. While extending the enum, put new values in alphabetical order
  */
 public enum TransformationFieldsConfig {
+
   //Common Fields
   ID("id", "$.{recordType}.id"),
   HR_ID("hrid", "$.{recordType}.hrid"),
-  IDENTIFIERS("identifiers", "$.{recordType}[*].identifiers[?(@identifierTypeId=={id})].value", IDENTIFIER_TYPES),
+
   SOURCE("source", "$.{recordType}.source"),
   METADATA_CREATED_DATE("metadata.createdDate", "$.{recordType}.metadata.createdDate", MetadataParametersConstants.getFixedLengthDataElement()),
   METADATA_UPDATED_DATE("metadata.updatedDate", "$.{recordType}.metadata.updatedDate"),
   METADATA_CREATED_BY_USER_ID("metadata.createdByUserId", "$.{recordType}.metadata.createdByUserId"),
   METADATA_UPDATED_BY_USER_ID("metadata.updatedByUserId", "$.{recordType}.metadata.updatedByUserId"),
 
   //Instance specific fields
+  ALTERNATIVE_TITLES("alternativeTitles", "$.instance[*].alternativeTitles[?(@.alternativeTitle=={value})]", ALTERNATIVE_TITLE_TYPES),
```

Comment:
I know you showed me the example yesterday Andrii, but comparing alternative tile is not the right way, atleast from UI the typeId is mandatory. If that is not present we can ignore that mapping, but this must be aletrnativeTitleTypeId

Reply:
because the title is record specific it will never match anything

Reply:
ok, updated